datasets/census_bureau_international/_images/run_csv_transform_kub/requirements.txt

```diff
@@ -0,0 +1,3 @@
+requests
```

Comment:
Remove request as you are pulling data using gsutil...

Reply:
done